src/HumanView.cpp

```diff
@@ -31,30 +31,83 @@ void HumanView::update(float time) {
     
 }
 
+
 void HumanView::drawObjects() {
     display.clear();
-    if (!texture.loadFromFile("../res/chad_sized.png"))
+
+    display.draw(logic.getPlayer().getSprite());
+    for(int i = 0; i < logic.getActors().size(); ++i)
     {
-        // error...
+        display.draw(logic.getActors()[i].getSprite());
     }
-    sf::Sprite player(texture);
-    player.setPosition(width/2, height/2);
-    display.draw(player);
     display.display();
 }
 
 void HumanView::checkKeyboard(float time) {
+    time = (time < 1.f) ? 1.f : time;
     if (sf::Keyboard::isKeyPressed(right)) {
         //character moves right
+        if (logic.getPlayer().isInAir())    //character has less directional control when in the air
```

Comment:
This whole section should probably be in the GameLogic  class itself, so that HumanView just calls a playerMoveRight() method rather than performing the checks in the HumanView (since that counts as logic and should not happen in the HumanView)